package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@foxglove/rosmsg",
   "version": "4.0.0",
-  "description": "ROS1 and ROS2 message definition parser",
+  "description": "ROS1, ROS2 and ROS2IDL message definition parser",
```

Comment:
Minor nit: ros2idl is a schema encoding while the other two are not. Technically those are `ros1msg1` and `ros2msg` in the mcap spec: https://mcap.dev/specification/appendix.html#ros1msg

So not sure what this description is trying to say here since ROS 1 and ROS 2 are the frameworks

Reply:
We could say `Parser for ROS .msg and .idl definitions`?